### PR TITLE
Added PT-BR translation

### PR DESCRIPTION
**Pull request by MatheusMuriel**
Required semantic review

### CODE REVIEW

File: data/pt-br/choices.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+module.exports = {
+  plus: [
+    {Impreciso
```
Comment (zrrrzzt) [inline data/pt-br/choices.js]: This is not valid. Please move The word Impreciso before ```text: ```

**Comment by zrrrzzt** [comment]
Thank you for the PR @MatheusMuriel :-)

It would be nice if you fixed the error I commented on and then I think we´re ready to merge

**Comment by MatheusMuriel** [comment]
> 
> 
> Thank you for the PR @MatheusMuriel :-)
> 
> It would be nice if you fixed the error I commented on and then I think we´re ready to merge

I'm sorry, it was a typo

**Comment by MatheusMuriel** [comment]
Still giving an error?

File: data/languages.json [hunk 2]
```diff
@@ -1,3 +1,4 @@
 [
   "en"
```
Comment (zrrrzzt) [inline data/languages.json]: You'll need a comma `,`to separate the options in languages.json

**Comment by MatheusMuriel** [comment]
\o/

**Comment by zrrrzzt** [comment]
Merged. Thx again

**Pull request merged**